tests/toranj/build.sh

```diff
@@ -174,86 +168,6 @@ case ${build_config} in
         cp -p ${top_builddir}/examples/apps/ncp/ot-ncp-ftd ${top_builddir}/examples/apps/ncp/ot-ncp-ftd-15.4-trel
         ;;
 
-    rcp)
```

Comment:
Suggest keeping the rcp build option here.   use it a lot during development and testing (quick way to conifigure a build with rcp only).

Reply:
I'm fine with keeping this part.